### ISSUE TITLE
Freeze Panes Tool Not Working in Spreadsheets

### ISSUE THREAD

**Issue by pvstyps** [standardnotes/forum#3813]
The "freeze panes" feature isn't working for me in spreadsheets. I click on the rows or columns I want to freeze, go to "freeze panes" and click on the appropriate option. The status update icon shows the spinning arrows and then the green arrow, indicating a change has been made and saved, but no rows or columns are frozen. It seems to not do anything at all. I'd like to be able to keep certain rows and colums frozen in some of my spreadsheets. Is anyone else having this issue? This is happening on the desktop version (3.195.13) on Windows 10.

**Comment by leenipper** [comment]
I observe the same not working for freeze panes in the web app version.
I actually saw it work one time, but then when attempting to freeze two rows at once, it did not work. And I cannot seem to make a single row freeze now.

My steps are:
1. choose the row(s) to freeze by clicking on the row number or selecting a range of cells for multiple rows.
2. click 3 dots menu on right and choose freeze panes
3. from the options shown, click on the freeze rows
4. I then scroll the sheet up to check if the applicable row stays in place on screen. It does not.

Are these steps correct ?

**Comment by VinceD53** [comment]
Steps are correct but it is not working as of this date (I just purchased this, so hopefully support is responsive.)

**Comment by michaelbeijer** [comment]
Indeed, I am on the professional trial, but freezing rows does not work at all. I hope they fix this before my trial runs out!

**Comment by leenipper** [comment]
Any update on this bug?
